### DISCUSSION TITLE
map key to change window title

### DISCUSSION THREAD

**Discussion by kennycallado** [kovidgoyal/kitty#3424] [Q&A]
Hello,

I was wondering if it is possible map a key to change the current window title. I know that there is a way to do it with the next command:

`kitty @ set-window-title Editor`

However I would like to do it from a keybinding like:

`launch --title "Titulo"`

Thanks in advance and congratulations on such good proyect!

**Comment by kovidgoyal** [comment]
Use remote control which you can map to key bindings as described https://sw.kovidgoyal.net/kitty/remote-control.html#mapping-key-presses-to-remote-control-commands

**Reply by kennycallado** [reply]
Ok, it works. Thank you 👍
